lib/game/universe.go

```diff
@@ -10,11 +10,12 @@ const (
 )
 
 type Universe struct {
-	height   uint32
-	width    uint32
-	cells    []uint8
-	newCells []uint8
-	stable   bool
+	height     uint32
+	width      uint32
+	cells      []uint8
+	newCells   []uint8
+	stable     bool
+	Generation uint32
```

Comment:
I don't think we are currently using this, right? Is there any plan to or can we remove it?

Reply:
I am planning to use it for an upcoming feature, which is why I added it.